features/support/tag_helpers.rb

```diff
@@ -1,3 +1,28 @@
+# Creates a tag with the given name in the local repo
+def create_local_tag tag_name
+  p "CREATING LOCAL TAG"
+  run "git tag -a #{tag_name} -m '#{tag_name}'"
+end
+
+
+# Creates a tag with the given name in the remote repo
+def create_remote_tag tag_name
+  p "CREATING REMOTE TAG"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.